lib/install/management/appliance.go

```diff
@@ -504,29 +498,27 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 	var info *types.TaskInfo
 
 	// Create the VCH inventory folder
-	vchFolder := d.session.VMFolder
```

Comment:
if we are not targeting a VC then the VCHFolder should be the `VMFolder`. Removing this line may will likely cause problems when trying to create targeting a standalone esxi

Reply:
Session defaults to `VmFolder` for `VCHFolder` so shouldn't be an issue

Reply:
Ah I just saw this as well. 👍